src/test/java/com/fullteaching/e2e/no_elastest/common/BaseLoggedTest.java

```diff
@@ -224,19 +243,21 @@ private void login(BrowserUser user, String userEmail, String userPass,
         try {
             userName = getUserName(user, true, APP_URL);
         } catch (NotLoggedException | ElementNotFoundException e) {
-            // TODO Auto-generated catch block
-            e.printStackTrace();
+            if (e.getClass().isInstance(NotLoggedException.class)){
+            log.error("The user {} was not logged",user.getClientData());}
+            else{
+                log.error("The userName field not found" );
+            }
```

Comment:
Shouldn't this make the test fail?

On the other hand, is it really necesary to control the exceptions? There are two diffferent named exceptions, each represents a cause of failure, what if execute `userName = getUserName(user, true, APP_URL);` without any exception control, and let JUnit control the exceptions to fail the test with an error?